charts/dbildungscloud-iam-keycloak/Chart.yaml

```diff
@@ -4,4 +4,4 @@ version: 0.1.1
 
 description: dbildungs-iam-keycloak
 type: application
-#appVersion: "latest"  # (see :https://github.com/dBildungsplattform/dbildungs-iam-keycloak/pkgs/container/dbildungs-iam)
\ No newline at end of file
+AppVersion: ""
```

Comment:
I think this should be `appVersion` (despite being referenced as `.Chart.AppVersion`.